packages/@aws-cdk/assertions/package.json

```diff
@@ -108,7 +107,18 @@
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
   "nozem": {
-    "ostools": ["dirname", "cd", "bash", "rm", "xargs", "sed", "mkdir", "rsync", "cat", "find"],
+    "ostools": [
```

Comment:
Any chance this can be reverted? In the past, this type of change has caused forward merge failures.

Reply:
Sure. pkglint auto fixup does this automatically.